### PR TITLE
fix: return 409 in case of duplicate

### PR DESCRIPTION
**Pull request by sstrigler**
Fixes [EMQX-7992](https://emqx.atlassian.net/browse/EMQX-7992)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/` dir
- [x] For internal contributor: there is a jira ticket to track this change

### CODE REVIEW

File: changes/v5.0.12-zh.md [hunk 1]
```diff
@@ -25,3 +25,5 @@
 - 在遥测功能未开启时，通过 /telemetry/data 请求其数据，将会返回 404 [#9464](https://github.com/emqx/emqx/pull/9464)。
 
 - 修复了一些 MQTT 协议包的潜在解析错误 [#9477](https://github.com/emqx/emqx/pull/9477)。
+
+- [FIXME] `POST /traces` to return `409` in case of duplicate [#9494](https://github.com/emqx/emqx/pull/9494).
```
Comment (sstrigler) [inline changes/v5.0.12-zh.md:29]: @lafirest please translate! 🍪

Comment (lafirest) [reply changes/v5.0.12-zh.md:29]: 现在，当重复调用 `POST /traces` 时，将会返回 `409` ，而不再是 `400` [#9494](https://github.com/emqx/emqx/pull/9494)。

**Review by thalesmg — APPROVED** [review]

File: changes/v5.0.12-zh.md [hunk 2]
```diff
@@ -39,3 +39,5 @@
 - 通过 `emqx_authenticator` 修复隐藏 `'client.authenticate'` 回调。 现在 `emqx_authenticator` 如果没有任何验证器匹配，则将执行传递给进一步的回调 [#9496](https://github.com/emqx/emqx/pull/9496)。
 
 - 如果在调用 `/trace/:id/download?node={node}` 时，`node` 不存在，则会返回 `400` [#9478](https://github.com/emqx/emqx/pull/9478).
+
+- 现在，当重复调用 `POST /traces` 时，将会返回 `409` ，而不再是 `400` [#9494](https://github.com/emqx/emqx/pull/9494)。
```
Comment (zmstone) [inline changes/v5.0.12-zh.md:43]:
```suggestion
- 当重复调用 `POST /traces` 时，将会返回 `409` ，而不再是 `400` [#9494](https://github.com/emqx/emqx/pull/9494)。
```

File: changes/v5.0.12-zh.md [hunk 3]
```diff
@@ -39,3 +39,5 @@
 - 通过 `emqx_authenticator` 修复隐藏 `'client.authenticate'` 回调。 现在 `emqx_authenticator` 如果没有任何验证器匹配，则将执行传递给进一步的回调 [#9496](https://github.com/emqx/emqx/pull/9496)。
 
 - 如果在调用 `/trace/:id/download?node={node}` 时，`node` 不存在，则会返回 `400` [#9478](https://github.com/emqx/emqx/pull/9478).
```
Comment (zmstone) [inline changes/v5.0.12-zh.md:41]:
```suggestion
- 如果在调用 `/trace/:id/download?node={node}` 时，`node` 不存在，则会返回 `400` [#9478](https://github.com/emqx/emqx/pull/9478)。
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**